pkg/bearertoken/grpc.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2024 The Jaeger Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package bearertoken
+
+import (
+	"context"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/metadata"
+)
+
+const BearerTokenKey = "bearer.token"
```

Comment:
```suggestion
const Key = "bearer.token"
```
Otherwise it will stutter `bearertoken.BearerTokenKey`